examples/src/main/scala/libretto/examples/canteen/Protocol.scala

```diff
@@ -2,6 +2,7 @@ package libretto.examples.canteen
 
 import libretto.scaletto.StarterKit._
 import libretto.scaletto.StarterKit.$._
+import libretto.scaletto.StarterKit.scalettoLib.given // FixMe! Why do we need this?
```

Comment:
Yeah, there is something messed up with how the imports work.

The part needs imho redesign.